mlrun/runtimes/nuclio/application/application.py

```diff
@@ -607,6 +607,11 @@ def _ensure_reverse_proxy_configurations(function: RemoteRuntime):
             spec, "spec.build.functionSourceCode"
         )
         function.spec.nuclio_runtime = mlrun.utils.get_in(spec, "spec.runtime")
+        logger_level_key = "spec.loggerSinks"
```

Comment:
Add a comment that this changes the reverse proxy log level, because it's not super straightforward